fe/fe-core/src/main/java/com/starrocks/statistic/sample/ColumnStats.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.statistic.sample;
+
+import com.starrocks.catalog.Type;
+import org.apache.commons.lang.StringEscapeUtils;
+
+public abstract class ColumnStats {
```

Comment:
consider the FullStatistic, can also use it well

Reply:
yes, we will optimize the full statistic collect job in the next pr.